Update XAMPP download archive (#1460)
contao/docs: docs/manual/guides/local-installation/xampp.en.md

This tutorial describes the local use of Contao with [XAMPP](https://www.apachefriends.org/) for Windows. 

> We use a »XAMPP Portable Version«, which only needs to be copied. You select the appropriate [Windows archive](https://www.apachefriends.org/download.html).

Reviewer: ```suggestion
We use a »XAMPP Portable Version«, which only needs to be copied. Download the appropriate [Windows archive](https://www.apachefriends.org/download.html) for this purpose.
```
